internal/pkg/cli/run_local.go

```diff
@@ -656,11 +680,27 @@ func (o *runLocalOpts) watchLocalFiles(stopCh <-chan struct{}) (<-chan interface
 					}
 				}
 
-				// TODO(Aiden): implement dockerignore blacklist for update
-				if !isHidden {
+				// skip updates from files matching .dockerignore patterns
+				isExcluded := false
+				for _, pattern := range o.dockerExcludes {
```

Comment:
I think it is possible (and perfectly reasonable) for `copilot/` to be in .dockerignore, though we probably want to keep watching `copilot/` even if it's supposed to be ignore by docker.